src/Sylius/Behat/Context/Api/Shop/LocaleContext.php

```diff
@@ -85,9 +90,23 @@ public function theLocaleWithCodeShouldNotBeAvailable(string $name, string $code
      */
     public function iShouldHaveWithCode(string $name, string $code): void
     {
-        $response = $this->client->getLastResponse();
+        $response = $this->localesClient->getLastResponse();
 
         Assert::true($this->responseChecker->hasValue($response, 'name', $name));
         Assert::true($this->responseChecker->hasValue($response, 'code', $code));
     }
+
+    /**
+     * @Then I should shop using the :localeCode locale
+     * @Then I should still shop using the :localeCode locale
+     */
+    public function iShouldShopUsingTheLocale(string $localeCode)
+    {
+        $this->cartsClient->buildCreateRequest();
+
+        Assert::same(
+            $this->responseChecker->getValue($this->cartsClient->create(), 'localeCode'),
```

Comment:
I know that we talked about it on audio but the default locale shouldn't be taken from the cart (we should get it from the channel).

Reply:
Indeed, that makes sense now for me, so I'll keep it as it is.